src/App.jsx

```diff
@@ -1,11 +1,37 @@
-import './App.css'
+import "./App.css";
+
+import { Routes, Route } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+import Header from "./head-nav/Header";
+import Nav from "./head-nav/nav";
+import ListArticlesView from "./articles-view/ListArticlesView";
+import SingleArticleView from "./single-article-view/SingleArticleView";
+
+import { getFromApi } from "./utils/utils";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [articles, setArticles] = useState([])
```

Comment:
I think this state could possibly be lower?  Though it would depend on your plan.  Your call, but have a think about it!

Reply:
True, i was wanting to have it accessable by category view but use different util functions to change it, I'll see what happens when I get to categories.